framework/auth/views.py

```diff
@@ -199,10 +199,10 @@ def auth_logout(redirect_url=None):
     return resp
 
 
-def auth_email_logout(token, user):
+def auth_email_logout(token, user, redirect_url=None):
     """When a user is adding an email or merging an account, add the email to the user and log them out.
     """
-    redirect_url = web_url_for('auth_login')
+    redirect_url = cas.get_login_url(redirect_url)
```

Comment:
Oh, ok, maybe it does handle `None`. Yeah, I think setting redirect_url to `/` makes sense in the call to `get_login_url` makes sense.